less/plugins/adapt-contrib-boxmenu/boxMenuItem.less

```diff
@@ -52,6 +58,14 @@
     color: @disabled-inverted;
   }
 
+  &__status {
+    margin-left: @menu-item-status-margin;
+  }
+
+  &.is-locked &__status .icon {
+    .icon-padlock-locked;
+  }
```

Comment:
This feels functional, should it not be in the boxMenu?

Reply:
Thanks for the call-out. I have adjusted this.